src/timeLine.ts

```diff
@@ -1393,6 +1399,9 @@ export class Timeline implements powerbiVisualsApi.extensibility.visual.IVisual
 
         this.renderTimeRangeText(timelineData, settings.rangeHeader);
 
+        // reset scroll position
```

Comment:
Why we need this?

Reply:
When we change the granularity the visual triggers update event. Already existing header element has `transform="translate(x, 0)" value, meaning the scroll is not reset by default.

It's apparent no the screenshot below
<img width="1757" alt="image" src="https://github.com/user-attachments/assets/e5cd17a1-2f15-44c1-bc92-2ae75a29af15">